src/index.js

```diff
@@ -2908,6 +2946,191 @@ const initializeFormElements = () => {
     }
   };
 
+  /**
+   * Send With Invalid Value
+   */
+
+  sendWithInvalidValue.onclick = async () => {
+    try {
+      const from = accounts[0];
+      const send = await provider.request({
+        method: 'eth_sendTransaction',
+        params: [
+          {
+            from,
+            to: '0x0c54FcCd2e384b4BB6f2E405Bf5Cbc15a017AaFb',
+            value: 'string',
```

Comment:
[non-blocker] i think this could be helpful as a comment in the code

Reply:
i think this could be helpful to add these github comments to comments directly in the code, or somehow update the values to be more explicit to their intention